[2024-07-02T23:15:44.604+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ecomm_data_pipeline.generate_data manual__2024-07-02T23:10:56+00:00 [queued]>
[2024-07-02T23:15:44.606+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ecomm_data_pipeline.generate_data manual__2024-07-02T23:10:56+00:00 [queued]>
[2024-07-02T23:15:44.607+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-07-02T23:15:44.611+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): generate_data> on 2024-07-02 23:10:56+00:00
[2024-07-02T23:15:44.612+0000] {standard_task_runner.py:57} INFO - Started process 286 to run task
[2024-07-02T23:15:44.614+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'ecomm_data_pipeline', 'generate_data', 'manual__2024-07-02T23:10:56+00:00', '--job-id', '84', '--raw', '--subdir', 'DAGS_FOLDER/ecomm_dag.py', '--cfg-path', '/tmp/tmp88bsgjnt']
[2024-07-02T23:15:44.614+0000] {standard_task_runner.py:85} INFO - Job 84: Subtask generate_data
[2024-07-02T23:15:44.634+0000] {task_command.py:410} INFO - Running <TaskInstance: ecomm_data_pipeline.generate_data manual__2024-07-02T23:10:56+00:00 [running]> on host 5b8fb0180af8
[2024-07-02T23:15:44.668+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='ecomm_data_pipeline' AIRFLOW_CTX_TASK_ID='generate_data' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T23:10:56+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-02T23:10:56+00:00'
[2024-07-02T23:15:44.668+0000] {logging_mixin.py:149} INFO - Running data generation script...
[2024-07-02T23:16:35.592+0000] {taskinstance.py:1093} INFO - Dependencies not met for <TaskInstance: ecomm_data_pipeline.generate_data manual__2024-07-02T23:10:56+00:00 [running]>, dependency 'Task Instance State' FAILED: Task is in the 'running' state.
[2024-07-02T23:16:35.592+0000] {taskinstance.py:1093} INFO - Dependencies not met for <TaskInstance: ecomm_data_pipeline.generate_data manual__2024-07-02T23:10:56+00:00 [running]>, dependency 'Task Instance Not Running' FAILED: Task is in the running state
[2024-07-02T23:16:35.592+0000] {local_task_job_runner.py:154} INFO - Task is not able to be run
